src/Schema/Validation/IdValidation.php

```diff
@@ -24,7 +24,7 @@ protected function validateDuplicities(SchemaBuilder $builder): void
 			foreach ($controller->getMethods() as $method) {
 
 				// Skip if @Id is not set
-				if (empty($method->getId())) continue;
```

Comment:
Why?

Reply:
Trying to throw away all `empty()`, it's too magical.
E.g. `empty('0') === true`. And `'0'` is valid id.

Reply:
I see